src/platform/lumin-runtime/elements/builders/image-builder.js

```diff
@@ -130,6 +139,35 @@ export class ImageBuilder extends UiNodeBuilder {
     }
   }
 
+  _createSpinner (prism) {
+    const spinner = this._createNode(ui.UiLoadingSpinner, 'Create', prism, ui.LoadingSpinnerType.k2dSpriteAnimation);
+    const [w, h] = spinner.getSize();
```

Comment:
Should the spinner size depend on image size? 🤔

Reply:
My intention was to show the user that something is happening. As long as it is well visible to the user it should be okay. However, if there is a clear need to depend on the image size we can always update the code.